README.md

```diff
@@ -1,25 +1,175 @@
+
+  STEERING_CHANNEL = 12           #channel on the 9685 pwm board 0-15
+  STEERING_LEFT_PWM = 739         #pwm value for full left steering
+  STEERING_RIGHT_PWM = 400        #pwm value for full right steering
+  
+  STEERING_PWM_PIN = 12           #Pin numbering according to Broadcom numbers
+  STEERING_PWM_FREQ = 75          #Frequency for PWM
+  STEERING_PWM_INVERTED = False   #If PWM needs to be inverted
+
+  THROTTLE_CHANNEL = 13           #channel on the 9685 pwm board 0-15
+  THROTTLE_FORWARD_PWM = 575      #pwm value for max forward throttle max 750
+  THROTTLE_STOPPED_PWM = 470      #pwm value for no movement
+  THROTTLE_REVERSE_PWM = 400      #pwm value for max reverse throttle
+
+  THROTTLE_PWM_PIN = 13           #Pin numbering according to Broadcom numbers
+  THROTTLE_PWM_FREQ = 75          #Frequency for PWM
+  THROTTLE_PWM_INVERTED = False   #If PWM needs to be inverted
+
+  AUTO_RECORD_ON_THROTTLE = False #if true, we will record whenever throttle is not zero. if false, you must manually toggle recording with some other trigger. Usually circle button on joystick.
+  CONTROLLER_TYPE='custom' # Set the controller to be used to be our custom one ()
+
+  LOGGING_LEVEL='DEBUG'
+  ```
+
+## 🚀 Launch ansible recipe
+
+***
+
+After have create your ansible playbook, you can launch it with these two options :
+- install it by ssh but you should configure manually internet, ssh and install ansible on your raspberry pi. Follow this [tutorial](ansible/README.md)
+- generate your raspian image with donkeycar installed, follow this [tutorial](packer/README.md)
+
+  <br/>	
+</details>
+
+<details>	
+  <summary><b>✨ Our features</b></summary>
+  <br/>	
+ 
+## Software 
+
+***
+
+### Ansible remote installation documentation : 
+  > we have made a ansible book for automate our donleycar and raspberry pi installation remotely (ssh).
+    Indeed, these recipes allow us to have one single clean installation and be able to repeat it to infinity. <br>
+    <b>ℹ️ Read [documentation](ansible/README.md) !! </b>
+
+### Generate raspian image with donkeycar installation :
+ > With ansible and packer, we can also generate a raspian image with our donkeycar installation. This feature is very usefull if we want to share it with people and don't use ssh. <br>
+ <b>ℹ️ Read [documentation](ansible/README.md) !! </b>
+
+### Generate raspian image with donkeycar installation :
+ > With ansible and packer, we can also generate a raspian image with our donkeycar installation. This feature is very usefull if we want to share it with people and don't use ssh. <br>
+ <b>ℹ️ Read [documentation](packer/README.md) !! </b>
+
+### Donkeycar manager :
+ > We have created a donkeycar manager. This manager is separated in to parts : a backend ( with database, api ...) and a frontend ( vuejs 2), it allow us to see cars who running, stopped, manage players who race... <br>
```

Comment:
to see cars which are running ...
manage players who race -> manage players in races